contributing-guides/maintainers-guide.md

```diff
@@ -56,35 +56,45 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**,
-  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
-  (4) have been open for at least **24 hours** unless the changes are trivial
-
-- If a PR fails to get a review from a second maintainer after a few days,
-  the first maintainer should ping others for review. If it still lingers around
+  (2) have all the **review comments addressed**,
+  (3) get **approved reviews by at least two maintainers**, (the second maintainer can merge immediately after approving) and
+  (4) have been open for at least **12 hours** unless the changes are trivial (contains only typo fixes or shell builtin/keyword/quoting syntax fixes)
```

Comment:
```suggestion
  (4) have been open for at least **12 hours** unless the changes are trivial ( E.G. contains only typo fixes or syntax error fixes)
```

There is just no need to so specific, I think it can be left the maintainers digression.

Reply:
I've written that it's up to maintainers to decide what is trivial or not. It means that everything depends on reviewers who review some PR.